### PR TITLE
Voting contest section

### LINKED ISSUE #36: Voting contest page
We need a page with a full description of the voting contest rules, procedure and prizes! :rocket:

### PR DESCRIPTION
**Pull request by ivov**
Closes #36

### CODE REVIEW

**Comment by ivov** [comment]
Merge conflict solved for imports and routes in `App.js`.

File: web/src/App.js [hunk 1]
```diff
@@ -61,6 +62,9 @@ class App extends React.Component {
             <Route path="/events" component={() =>
               <Events accessToken={this.state.accessToken} events={this.state.events} />
             } />
+            <Route path="/voting-contest" render={() =>
```
Comment (shu8) [inline web/src/App.js]: I think we could simplify this to the `<Route path="/voting-contest" component={VotingContest} />` style route

**Review by shu8 — APPROVED** [review]
this looks awesome!

**Comment by shu8** [comment]
Merge conflicts resolved

**Pull request merged**